src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Controllers/PublicationControllerTests.cs

```diff
@@ -100,6 +104,45 @@ public async Task GetPublicationTree()
             Assert.Single(topic.Publications);
         }
 
+        [Fact]
+        public async Task ListSitemapItems()
+        {
+            var publicationService = new Mock<IPublicationService>(Strict);
+
+            publicationService.Setup(mock => mock.ListSitemapItems())
+                .ReturnsAsync(new List<PublicationSitemapItemViewModel>()
+                {
+                    new()
+                    {
+                        Slug = "test-publication",
+                        LastModified = DateTime.Parse(publicationLastModifiedTime),
+                        Releases =
+                        [
+                            new ReleaseSitemapItemViewModel
+                            {
+                                Slug = "test-release",
+                                LastModified = DateTime.Parse(sitemapItemLastModifiedTime)
+                            }
+                        ]
+                    }
+                });
+
+            var controller = BuildPublicationController(publicationService: publicationService.Object);
+
+            var response = await controller.ListSitemapItems();
+            var sitemapItems = response.AssertOkResult();
+
+            var item = Assert.Single(sitemapItems);
+            Assert.Equal("test-publication", item.Slug);
+
+            Assert.NotNull(item);
+            Assert.NotNull(item.Releases);
```

Comment:
`NotNull` checks don't really add anything here as non-null is inferred by other assertions.

Reply:
Technically, you're right, but because we're in a `#nullable` context here `item.Releases` generates a possibly null warning unless it's explicitly checked first. Frustrating, I know!